interface/client/lib/helpers/helperFunctions.js

```diff
@@ -50,6 +50,21 @@ Helpers.formatUrl = function(url){
 };
 
 /**
+Sanatizes URLs to prevent phishing and XSS attacks
+
+@method sanatizeUrl
+@param {String} url
+**/
+Helpers.sanatizeUrl = function(url){    
+    // add http:// if no protocol is present
+    if(_.isString(url)) {
+        url = url.replace(/^(javascript:)?(data:)?\/\//i, 'http:');
+    }
+
+    return url;
```

Comment:
blacklist is not sufficient. `javaScript://` is also valid, and maybe special characters such as %0A . 

Also, why only handle strings? 
If you expect anything other than strings, such as location-objects, you should filter them. If you expect string input, then cast the url to string and do the sanitation unconditionally.